src/views/ProfessorHome.vue

```diff
@@ -16,6 +16,14 @@ export default {
     ...mapGetters([
       'getUser'
     ])
+  },
+
+  methods: {
+    createSection() {
```

Comment:
I think a more appropriate function name would be goToCreateSection?

Reply:
Yes that does sound better since we're not actually creating a section but rather redirecting to the page.